### PR TITLE
Cult_Contact_Updated

### PR DESCRIPTION
**Pull request by RaviKumar7870**

### CODE REVIEW

**Comment by RaviKumar7870** [comment]
Contact page detail changed

File: contacts.html [hunk 1]
```diff
@@ -712,9 +679,9 @@ <h2>Contacts</h2>
 						</tr>
 						<tr>
 							<td class='tg-i6ua'>Drama Club<br></td>
```
Comment (Signior-X) [inline contacts.html:681]: Add three more in Dance club
Samarth Neema b19112
Shivani Pandey b19266
Aarushi Gajri b19001

File: contacts.html [hunk 2]
```diff
@@ -657,50 +640,34 @@ <h2>Contacts</h2>
 							<td class='tg-03to'></td>
 						</tr>
 						<tr>
-							<td class='tg-i6ua'>Shutterbugs(Photography Club)<br></td>
-							<td class='tg-03to'>Satyam Shukla<br></td>
-							<td class='tg-03to'>b17058@students.iitmandi.ac</td>
-							<td class='tg-03to'>9630262761</td>
+							<td class='tg-i6ua'>PMC<br></td>
```
Comment (Signior-X) [inline contacts.html:643]: Instead of PMC, write "Photography and Moviemaking club (PMC)"

**Review by Signior-X — CHANGES_REQUESTED** [review]

**Comment by Signior-X** [comment]
Nice work @RaviKumar7870 , just make some small changes.

**Comment by Signior-X** [comment]
One suggestion, in the PR description, you can write 
Fixes #22 or mention some issue number like #issue-number to show a relationship.

**Review by RaviKumar7870 — COMMENTED** [review]
Fixes #22

**Review by RaviKumar7870 — COMMENTED** [review]
Fixes #22

**Review by Signior-X — APPROVED** [review]

**Pull request merged**